src/pages/public_registration/action_accordions/DoiRequestAccordion.tsx

```diff
@@ -271,6 +258,17 @@ export const DoiRequestAccordion = ({
             </LoadingButton>
           </Box>
         )}
+
+        {isPendingDoiRequest && (
```

Comment:
Kanskje det burde legges in noen Dividere mellom de forskjellige segmentene?

Reply:
Muligens. Men uansett ett steg frem dette, så det er ingen blokker for meg :+1: 
Det gjelder sikkert andre ticket-typer også

Reply:
Jepp, kan ta det i en egen PR når det meste er på plass i alle accordions.